pyhf/__init__.py

```diff
@@ -380,7 +380,33 @@ def pvals_from_teststat(sqrtqmu_v, sqrtqmuA_v):
     return CLsb, CLb, CLs
 
 
-def runOnePoint(muTest, data, pdf, init_pars, par_bounds):
+def runOnePoint(muTest, data, pdf, init_pars = None, par_bounds = None):
+    r"""
+    Computes test statistics (and expected statistics) for a single value
+    of the parameter of interest
+
+    Args:
+        muTest (Number or Tensor): The value of the parameter of interest (POI)
+        data (Number or Tensor): The root of the calculated test statistic given the Asimov data, :math:`\sqrt{q_{\mu,A}}`
+        init_pars (Array or Tensor): the initial parameter values to be used for minimization
+        par_bounds (Array or Tensor): the parameter value bounds to be used for minimization
+
+    Returns:
+        Tuple of Floats: a tuple containing (qmu, qmu_A, CLsb, CLb, CLs, CLs_exp)
+                         where qmu and qmu_A are the test statistics for the
+                         observerd and asimov datasets respectivleyself.
```

Comment:
This looks good, and is nice to have. Also great to have a doc string for runOnePoint now. :+1: However, since this is going into the docs, I think there are some tiny typos here. I think this line should be "observed and Asimov datasets respectively."